assembly/assembly.xml

```diff
@@ -0,0 +1,27 @@
+<assembly xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Assembly files are conventionally placed under a `src/assembly` directory rather than at the top level next to the `pom.xml` file, so please create a `src/assembly` directory here and move this file there. Also, the name of the assembly file should indicate what it is assembling, so I suggest renaming the file to `standalone.xml`.